docs/nodejs-runtime/master.adoc

```diff
@@ -18,7 +18,7 @@ include::topics/dev-guide-mission-intro.adoc[leveloffset=+1]
 [[nodejs-available-missions-and-boosters]]
 == Available Missions and Boosters for {NodeJS}
 
-The {NodeJS} runtime supports the following missions and boosters.
+The {NodeJS} runtime has the following missions and boosters.
```

Comment:
The word _Has_ implies ownership, and the runtime can't really own things. I suggest using

_The following missions and boosters are available for the {NodeJS} missions:_

On a related note, why do we separate missions and boosters in this sentence? It's not factually correct, as it's actually just the boosters that we are talking about, or is it?

Reply:
@tradej @Ladicek  


> The following missions and boosters are available for the {NodeJS} missions

sounds a bit redundant (also, it makes no sense to say that we have missions available for missions), I'd go with something like:

> The following boosters are available for the {NodeJS} missions

alternatively:

> The following boosters are available for {NodeJS}

IMHO, there is little point in mentioning missions that do not have boosters available in a particular runtime. Therefore, saying there are boosters available for Node implies the missions that said boosters are base on are available, too.
Does that make sense or did I just spectacularly miss the point:?)

Reply:
The entire "launching space ships" metaphor just doesn't work all that well...

I agree with _The following boosters are available for {NodeJS}_, it's the simplest and also factually most correct.